packages/components/src/components/data-list/data-list.jsx

```diff
@@ -1,91 +1,170 @@
 import classNames from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
-import { VariableSizeList as List } from 'react-window';
 import { NavLink } from 'react-router-dom';
-import { isMobile } from '@deriv/shared';
+import { TransitionGroup } from 'react-transition-group';
+import { CellMeasurer, CellMeasurerCache } from 'react-virtualized/dist/es/CellMeasurer';
+import { AutoSizer } from 'react-virtualized/dist/es/AutoSizer';
+import { List } from 'react-virtualized/dist/es/List';
+import { isMobile, isDesktop } from '@deriv/shared';
 import DataListCell from './data-list-cell.jsx';
-import ThemedScrollbars from '../themed-scrollbars/themed-scrollbars.jsx';
-
-const ThemedScrollbarsWrapper = React.forwardRef((props, ref) => (
-    <ThemedScrollbars {...props} forwardedRef={ref}>
-        {props.children}
-    </ThemedScrollbars>
-));
-// Display name is required by Developer Tools to give a name to the components we use.
-// If a component doesn't have a displayName is will be shown as <Unknown />. Hence, name is set.
-ThemedScrollbarsWrapper.displayName = 'ThemedScrollbars';
+import ThemedScrollbars from '../themed-scrollbars';
 
 class DataList extends React.PureComponent {
```

Comment:
what is the difference between DataList and DataTable?

Reply:
Not much. `DataTable` is coupled with `TableRow` component. We can remove `DataTable` all together if we use  `DataList` instead and add `TableRow` component in the rowRenderer callback

Reply:
ok, is this worth doing now/later?

Reply:
I prefer to do this in a separate card

Reply:
allright 👍